src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -245,7 +246,8 @@ using MoreTypes = boost::variant<
         RBF<arma::mat, arma::mat, GaussianFunction>*,
         BaseLayer<GaussianFunction, arma::mat, arma::mat>*,
         PositionalEncoding<arma::mat, arma::mat>*,
-        ISRLU<arma::mat, arma::mat>*
+        ISRLU<arma::mat, arma::mat>*,
```

Comment:
Not sure yet, what the issue is, can you move `FastLSTM<arma::mat, arma::mat>*,` into `MoreTypes`, maybe that already solves the issue.